clubs/admin.py

```diff
@@ -47,4 +50,31 @@ class ClubAdmin(admin.ModelAdmin):
     list_filter = ('status', 'denial', 'is_active',)
     readonly_fields = (owner_email,)
 
+    def save_model(self, request, obj, form, change):
+        context = Context()
+        super(ClubAdmin, self).save_model(request, obj, form, change)
```

Comment:
I absolutely does not. You don't pass `this` around yourself, that's taken care of by the compiler. Instead you mark things either `static` as static class callable, or you don't mark them at all and then they're instance callable.